contributing.md

```diff
@@ -171,6 +171,9 @@ automatically apply EditorConfig files.
 
 - `*` and `&` bind to the *type*, e.g., `T* arg` instead of `T *arg`.
 
+- Type specifiers are specified in this order: `friend`, `static`, `inline`,
+  and `constexpr`.
```

Comment:
Type specifiers are are only types themselves, as in primitives, previously declared types, type aliases and so on. The specifiers are don't belong to any group. See https://en.cppreference.com/w/cpp/language/declarations#Specifiers for an exhaustive treatment.
We should also replace `static` with `<storage class>` and add `constinit` (at the end, i guess?).

Reply:
Fair point. I changed it up a bit.